docs/creating-bubbles.md

```diff
@@ -0,0 +1,51 @@
+# Creating Bubbles
+
+There may be situations where you want to have your own sub-models for your project. 
+This can allow you to toggle between different views and organize your logic for `Update`.
+
+## Showing Specific UI Components (Bubbles)
+In Bubbletea programs, you can decide which components are shown by holding a `state` field in your main model struct. 
```

Comment:
Careful with consistency — we should always spell _Bubble Tea_ as two words, both capitalized.